spp_scan_id_document/model/registrant.py

```diff
@@ -27,10 +27,13 @@ def on_scan_id_document_details(self):
                     vals = {
```

Comment:
@reichie020212 I think we should take this part out to be a hook method, so we can inherit that easier in the later modules

```python
def _prepare_update_vals(self, details):
    return {...}
```

then this line will become
```python
vals = self. _prepare_update_vals(details)
```

so, in later module, it will be:
```python
def _prepare_update_vals(self, details):
    res = super(). _prepare_update_vals(details)
    res.update(...)
    return res
```